<launch>

    <node name="ressac1" pkg="metal" type="hidden-ros.py" ns="ressac1"
        args="--plan $(find metal)/data/action-V.plan --executor=morse+ros --agentName=ressac1 --waitSignal" />

    <node name="ressac2" pkg="metal" type="hidden-ros.py" ns="ressac2"
        args="--plan $(find metal)/data/action-V.plan --executor=morse+ros --agentName=ressac2 --waitSignal" />

    <node name="mana" pkg="metal" type="hidden-ros.py" ns="mana"
        args="--plan $(find metal)/data/action-V.plan --executor=hyper --agentName=mana --waitSignal" />

    <node name="minnie" pkg="metal" type="hidden-ros.py" ns="minnie"
        args="--plan $(find metal)/data/action-V.plan --executor=morse+ros --agentName=minnie --waitSignal" />

</launch>
